### PR TITLE
Install spec helper and simplecov

### PR DESCRIPTION
**Pull request by NoahZinter**
98.9% coverage! 

Look for small merge conflicts here, this touches every .spec file. But all it does is remove the `require 'rspec'` at the head of each. Tests passing!

- [x] test passing
- [x] ready to merge

### CODE REVIEW

**Review by redferret — APPROVED** [review]
Amazing!!

File: spec/customer_spec.rb [hunk 1]
```diff
@@ -1,4 +1,3 @@
-require 'rspec'
```
Comment (mkrumholz) [inline spec/customer_spec.rb:1]: Love how much this cleans things up!

**Review by mkrumholz — APPROVED** [review]
Huzzah! ✨

**Review by sami-p — APPROVED** [review]
Wooo!

**Pull request merged**